### ISSUE TITLE
nanopolish

### ISSUE THREAD

**Issue by rditter** [ubccr/software-layer#226]
nanopolish-0.14.1
https://github.com/jts/nanopolish

**Comment by tonykew** [comment]
I have two test builds done:
nanopolish version 0.13.3 - uses the Fast5 library for fast5 files
nanopolish version 0.14.0 - uses the newer slow5lib library for SLOW files

The newer version does not include the Fast5 library, and there is no
indication that the slow5lib library reads fast5 files.

More info to follow...

**Comment by dsajdak** [comment]
@rditter nanopolish has been published to the `ccrsoft/2023.01` software repository.  Please see @tonykew's note above for details on the different versions.

```
vortex:~$ module spider nanopolish/0.13.3

----------------------------------------------------------------------------------------------------------------
  nanopolish: nanopolish/0.13.3
----------------------------------------------------------------------------------------------------------------
    Description:
      Software package for signal-level analysis of Oxford Nanopore sequencing data.

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "nanopolish/0.13.3" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    Help:

      Description
      ===========
      Software package for signal-level analysis of Oxford Nanopore sequencing data.


      More information
      ================
       - Homepage: https://github.com/jts/nanopolish





vortex:~$ module spider nanopolish/0.14.0

----------------------------------------------------------------------------------------------------------------
  nanopolish: nanopolish/0.14.0
----------------------------------------------------------------------------------------------------------------
    Description:
      Software package for signal-level analysis of Oxford Nanopore sequencing data.

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "nanopolish/0.14.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    Help:

      Description
      ===========
      Software package for signal-level analysis of Oxford Nanopore sequencing data.


      More information
      ================
       - Homepage: https://github.com/jts/nanopolish

vortex:~$  module load gcc/11.2.0  openmpi/4.1.1 nanopolish/version
```
